vermouth/processors/annotate_mut_mod.py

```diff
@@ -148,6 +149,8 @@ def _terminal_matches(resname, residue_graph, res_idx):
     neighbour = list(residue_graph[res_idx])[0]  # Only one neighbour by definition.
     resid = residue_graph.nodes[res_idx].get('resid', 0)
     neighbour_resid = residue_graph.nodes[neighbour].get('resid', 0)
+    if not is_protein(residue_graph.nodes[res_idx]['graph']):
```

Comment:
Just to make sure I understand right: this checks that the specific *residue* is protein. Right? If so I am happy.

Reply:
Correct.